pkgs/tools/text/mdbook-pdf/default.nix

```diff
@@ -3,23 +3,25 @@
 , fetchCrate
 , rustPlatform
 , pkg-config
+, rustfmt
 , openssl
 , CoreServices
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "mdbook-pdf";
-  version = "0.1.5";
+  version = "0.1.6";
 
   src = fetchCrate {
     inherit pname version;
-    hash = "sha256-822RQKgedxQ+VFNDv20tFUc2Xl56Ivub+/EXNrLRfGM=";
+    hash = "sha256-eaZ4LlswbVgC/pOZH3cevxGzlgKXFJnvcNMovqCEBOc=";
   };
 
-  cargoHash = "sha256-mX2EKjuWM1KW8DXFdYFKQfASjdqZCW78F1twZNQQr7o=";
+  cargoHash = "sha256-mj1heP231CWad6Uy4g9TCcDVj8f86Pf689kuePbzfBg=";
 
   nativeBuildInputs = [
     pkg-config
+    rustfmt
```

Comment:
Normally rustfmt is optionally and can be safely omitted?

Reply:
Unfortunately, this is needed to resolve the build error, because of the code here: https://github.com/mdrokz/auto_generate_cdp/blob/ae311b29a14c2e8380c40221e8b44e143979a311/src/lib.rs#L116

Reply:
Oh, I just found that maybe we can set the env to avoid this. https://github.com/mdrokz/auto_generate_cdp/pull/8

Reply:
![Screenshot from 2023-05-20 10-58-34](https://github.com/NixOS/nixpkgs/assets/43995067/39766986-810d-4dcd-a5f4-52644f67ff9e)

Looks like we still haven't bumped the version to https://github.com/mdrokz/auto_generate_cdp/pull/8 this PR, let's keep the `rustfmt` dependency for now.